src/pages/Authentication/Login.jsx

```diff
@@ -1,7 +1,86 @@
-import React from 'react'
+import axios from 'axios'
+import React, {useState} from 'react'
+import { Link, useNavigate } from 'react-router-dom'
+import { useAuth } from '../../contexts/AuthContext'
+import "./Login.css"
 
 export default function Login() {
+  const navigate = useNavigate()
+  const [loginData, setLoginData] = useState({email: "", password: ""})
+  const [loginError, setLoginError] = useState("")
+  const [showPwd, setShowPwd] = useState(false)
+  
+  const testUser = {
+    email: "theMarauders@gmail.com",
+    password:"FortunaMajor"
+  }
+
+  function setUserData(e){
+    const {name, value} = e.target
+    setLoginData(prev => ({...prev, [name]:value}))
+  }
+
+  const handleLogin = (e) => {
+    e.preventDefault()
+    sendLoginData(loginData)
+  }
+
+  const sendLoginData = async (loginData) =>{
+    try {
+      const response = await axios.post("/api/auth/login", loginData)
+      console.log(response)
+      if (response.status === 200){ 
+      const {data} = response;
+      const userToken = data.encodedToken
+      localStorage.setItem("userToken", userToken)
+      navigate("/products")
+      }
+    } catch (error) {
```

Comment:
Suggestion , Set the encodedToken in the auth Context , then use it wherever you want to use to fetch user data